js/src/dom/eventHandler.js

```diff
@@ -267,7 +257,19 @@ const EventHandler = (() => {
     },
 
     one(element, event, handler, delegationFn) {
-      addHandler(element, event, handler, delegationFn, true)
+      const isDelegation = typeof handler === 'string'
+      const realHandler  = isDelegation ? delegationFn : handler
+      function complete(e) {
+        EventHandler.off(element, e.type, complete)
+        realHandler.apply(element, [e])
+      }
+      complete.isOne = true
+      if (isDelegation) {
+        addHandler(element, event, handler, complete)
+      } else {
+        addHandler(element, event, complete)
+      }
+      realHandler.uid = complete.uid.toString()
```

Comment:
@Johann-S Adding the `uid` property to the real handler function will cause troubles when using and removing the same handler for more than one event (like in https://github.com/twbs/bootstrap/blob/v4-without-jquery/js/src/tooltip.js#L322).
It was changed in https://github.com/twbs/bootstrap/commit/fa90114c11212f1021ab22b5baf7201b4899f3a7 to take this into account, removing the setting of `uidEvent` property.
I thought that a test was written for this scenario, but i've probably missed it.

Reply:
Yep that's because `Util.noop` always return the same function instead of a new one here but yes in case of a reusable handler it will be problematic

Reply:
Yes, with `Util.noop` shouldn't be a problem 😄 
However, https://github.com/twbs/bootstrap/pull/24487 should fix that, but i think you have to revert your last commit in order to merge it. If you can reopen it, I'll submit a couple of tests to cover this case.